src/interactive.js

```diff
@@ -0,0 +1,23 @@
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+export function updateTaskStatus(task, status) {
+  task.completed = status;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function checkAllCompleted() {
+  const anyCompleted = tasks.some((task) => task.completed);
+  const clearAllButton = document.getElementById('clear-all');
+  clearAllButton.disabled = !anyCompleted;
+  clearAllButton.addEventListener('click', () => {
+    const completedTasks = tasks.filter((task) => task.completed);
+    const remainingTasks = tasks.filter((task) => !task.completed);
+    localStorage.setItem('tasks', JSON.stringify(remainingTasks));
+    completedTasks.forEach((task) => {
+      const index = tasks.indexOf(task);
+      tasks.splice(index, 1);
+    });
+    checkAllCompleted();
+    window.location.reload();
+  });
+}
```

Comment:
- [ ] [OPTIONAL] Please consider updating the tasks indices in the `localStorage` when a user clears all completed tasks.
![image](https://user-images.githubusercontent.com/46839548/229312951-d73959a2-d9bd-4bc4-b44c-aff0ce26401b.png)